doc/api/deprecations.md

```diff
@@ -807,10 +807,27 @@ Importing assert directly is not recommended as the exposed functions will use
 loose equality checks. Use `require('assert').strict` instead. The API is the
 same as the legacy assert but it will always use strict equality checks.
 
+<a id="DEP00XX"></a>
+### DEP00XX: timers.enroll()
+
+Type: Runtime
+
+`timers.enroll()` has been deprecated in favor of using the publicly documented timers APIs, namely [`setTimeout()`][] and [`setInterval()`][].
```

Comment:
Nit: Can we make this more concise? Maybe this?:

> `timers.enroll()` is deprecated. Please use [`setTimeout()`][] or [`setInterval()`][] instead.

(Similarly for the other deprecation message below.)